packages/wonder-blocks-clickable/util/types.js

```diff
@@ -0,0 +1,229 @@
+// @flow
+import * as React from "react";
+
+import type {CSSProperties} from "aphrodite";
+
+type NestedArray<T> = $ReadOnlyArray<T | NestedArray<T>>;
+type Falsy = false | 0 | null | void;
+export type StyleType =
+    | CSSProperties
+    | Falsy
+    | NestedArray<CSSProperties | Falsy>;
+
+// Source: https://www.w3.org/WAI/PF/aria-1.1/roles#role_definitions
+type roles =
+    | "alert"
+    | "alertdialog"
+    | "application"
+    | "article"
+    | "banner"
+    | "button"
+    | "checkbox"
+    | "columnheader"
+    | "combobox"
+    | "command" // abstract role
+    | "complementary"
+    | "composite" // abstract role
+    | "contentinfo"
+    | "dialog"
+    | "directory"
+    | "document"
+    | "form"
+    | "grid"
+    | "gridcell"
+    | "group"
+    | "heading"
+    | "img"
+    | "input" // abstract role
+    | "landmark" // abstract role
+    | "link"
+    | "list"
+    | "listbox"
+    | "listitem"
+    | "log"
+    | "main"
+    | "marquee"
+    | "math"
+    | "menu"
+    | "menubar"
+    | "menuitem"
+    | "menuitemcheckbox"
+    | "menuitemradio"
+    | "navigation"
+    | "note"
+    | "option"
+    | "presentation"
+    | "progressbar"
+    | "radio"
+    | "radiogroup"
+    | "range" // abstract role
+    | "region"
+    | "reletype" // abstract role
+    | "row"
+    | "rowgroup"
+    | "rowheader"
+    | "scrollbar"
+    | "search"
+    | "section" // abstract role
+    | "sectionhead" // abstract role
+    | "select" // abstract role
+    | "separator"
+    | "slider"
+    | "spinbutton"
+    | "status"
+    | "structure" // abstract role
+    | "tab"
+    | "tablist"
+    | "tabpanel"
+    | "textbox"
+    | "timer"
+    | "toolbar"
+    | "tooltip"
+    | "tree"
+    | "treegrid"
+    | "treeitem"
+    | "widget" // abstract role
+    // abstract role
+    | "window";
+
+type IdRef = string;
+type IdRefList = IdRef | Array<IdRef>;
+
+// Source: https://www.w3.org/WAI/PF/aria-1.1/states_and_properties
+// TODO(kevinb): convert to disjoint union of exact object types keyed on role
+// eslint-disable-next-line flowtype/require-exact-type
+export type AriaProps = {|
```

Comment:
e.g. If you needed it, you shouldn't need to export `AriaProps` here. Instead, you could import directly from `wonderblocks-core`.

Reply:
it actually wasn’t being used at all, I removed it